homeassistant/components/knx/expose.py

```diff
@@ -149,10 +149,6 @@ async def _async_entity_changed(self, event: Event) -> None:
 
     async def _async_set_knx_value(self, value: StateType) -> None:
         """Set new value on xknx ExposeSensor."""
-        if value is None:
```

Comment:
Keep the `None` guard.
`self.device.set()` can not be called with `None` (It's typed `Any` so mypy doesn't alert here.)